multicluster/PROJECT

```diff
@@ -0,0 +1,91 @@
+domain: crd.antrea.io
+layout:
+- go.kubebuilder.io/v3
+multigroup: true
+projectName: multicluster
+repo: antrea.io/antrea/multicluster
+resources:
+- api:
+    crdVersion: v1
+    namespaced: true
```

Comment:
ClusterClaim should not be namespaced right?

Reply:
Because we want to support dedicated leader (ie. a leader cluster manages multiple muli-cluster set), ClusterClaim is namespaces, where by default it is always in antrea-system.

Reply:
use Kube-system as default namespace for CRD instances and controller.

Reply:
done.